submit.go

```diff
@@ -77,15 +77,15 @@ type jsonLogEntry struct {
 	Lines string `json:"lines"`
 }
 
-
 type genericWebhookPayload struct {
 	parsedPayload
-	ReportURL string             `json:"report_url"`
-	ListingURL string            `json:"listing_url"`
+	ReportURL  string `json:"report_url"`
+	ListingURL string `json:"listing_url"`
 }
 
 // the payload after parsing
 type parsedPayload struct {
```

Comment:
it might be good to rename this type, and/or update the comment, since it's not longer really limited to the payload of the request.